core/src/main/java/com/novoda/noplayer/internal/exoplayer/forwarder/ErrorFormating.java

```diff
@@ -0,0 +1,12 @@
+package com.novoda.noplayer.internal.exoplayer.forwarder;
+
+final class ErrorFormating {
```

Comment:
I agree but this class feels like it's in the middle 😛, not quite a holder and not quite a formatter

if we had `interface Formating -> format()` and `ErrorFormating implements Formating` then I'd be 100% onboard, I guess that's pretty overkill for what we need 👻